qanary_commons/src/main/java/eu/wdaqua/qanary/communications/RestTemplateWithProcessId.java

```diff
@@ -19,7 +19,7 @@
 // A = Both, C = ProcessIdRestTemplate
```

Comment:
@dschiese Please add a comment here about why this condition exist and what the impact is.